pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -168,6 +167,13 @@ private String getDefaultBrokerId() {
     }
   }
 
+  @Override
+  public BrokerResponseNative handleRequest(JsonNode request, @Nullable RequesterIdentity requesterIdentity,
```

Comment:
Suggest adding `getResultTable()` into the `ResponseNative` interface, instead of relying on the implementation to return the `BrokerResponseNative`